### DISCUSSION TITLE
nuxt generate生成的静态站，页面没有尾斜杠会触发301重定向，如何不触发重定向

### DISCUSSION THREAD

**Discussion by againF** [nuxt/nuxt#10155] [Questions (Nuxt 2)]
访问https://example.com/folder 会重定向到

https://example.com/folder/

我想知道如何不触发这个重定向，因为在google seo优化时，谷歌会报重定向错误。

我想到两个解决方向：
【1】如何不触发重定向
【2】如何在生成路由的时候就是每个路由都是带尾斜杠的

**Comment by qyinter** [comment]
解决了吗好兄弟/。我这边就是这样但是看地址栏，他会把/去掉
